session.go

```diff
@@ -1781,6 +1790,28 @@ func (s *Session) NewBatch(typ BatchType) *Batch {
 	return batch
 }
 
+// Batch creates a new batch operation using defaults defined in the cluster
+func (s *Session) Batch(typ BatchType) *Batch {
+	s.mu.RLock()
+	batch := &Batch{
+		Type:             typ,
+		rt:               s.cfg.RetryPolicy,
+		serialCons:       s.cfg.SerialConsistency,
+		trace:            s.trace,
+		observer:         s.batchObserver,
+		session:          s,
+		Cons:             s.cons,
+		defaultTimestamp: s.cfg.DefaultTimestamp,
+		keyspace:         s.cfg.Keyspace,
+		metrics:          &queryMetrics{m: make(map[string]*hostMetrics)},
+		spec:             &NonSpeculativeExecution{},
+		routingInfo:      &queryRoutingInfo{},
+	}
+
```

Comment:
duplicate code (`NewBatch`)

Reply:
I have kept the `session.NewBatch` and marked it as deprecated according to your suggestion.

Reply:
Got it! Fixed.